website/docs/gator.md

```diff
@@ -427,6 +427,80 @@ However, not including the `namespace` definition in the call to `gator expand`
 error expanding resources: error expanding resource nginx-deployment: failed to mutate resultant resource nginx-deployment-pod: matching for mutator Assign.mutations.gatekeeper.sh /always-pull-image failed for  Pod my-ns nginx-deployment-pod: failed to run Match criteria: namespace selector for namespace-scoped object but missing Namespace
 ```
 
+## The `gator sync verify` subcommand
```

Comment:
Let's wait on editing this docs page further until the interface of the command is solidified.  Otherwise you could be doing a lot of time consuming work on editing this and then needing to do further changes.